build/build.go

```diff
@@ -683,8 +684,12 @@ func Build(ctx context.Context, drivers []DriverInfo, opt map[string]Options, do
 				resp[k] = res[0]
 				respMu.Unlock()
 				if len(res) == 1 {
+					digest := res[0].ExporterResponse["containerimage.digest"]
+					if opt.Quiet {
+						fmt.Println(digest)
```

Comment:
build pkg should not write to stdio. Read the value from the return params in command pkg.

Reply:
Yeah, I thought this felt wrong - I'll look at using the return value and update as you suggest.